.github/workflows/build_wheel.yml

```diff
@@ -3,6 +3,7 @@ name: cibuildwheel
 on:
   push:
     branches: [latest]
+    tags: v*
```

Comment:
(and this should fix building wheels on new tags, @ctb: try to force push `4.0.0a2` again after this is merged =])